app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,16 @@
+class PostsController < ApplicationController
+  skip_before_action :verify_authenticity_token
+
+  def index; end
+
+  def show
+    @post = Post.find params[:id]
+  end
+
+  def create
+    @post = Post.new(message: params[:msg])
+    @post.save
+
+    redirect_to '/posts'
```

Comment:
Try to avoid hard coded routes. Working within the Rails framework can avoid subtle bugs where if you changed the path of the controller it would automatically update these values. Try `redirect_to :action => :index`